### ISSUE TITLE
adding anther sandhi to the filter for final nasal variants

### ISSUE THREAD

**Issue by wujastyk** [chchch/upama#87]
I have frequent readings like  "kuṣṭhan tagaraṃ" that I'd like filtered out of the apparatus by the `final nasal variants` filter please.  All savarṇa nasals.

**Comment by chchch** [comment]
This is currently being handled by `final anusvāra variants (Malayālam)`, which looks for `<textLang mainLang="sa-Mlym"/>`. I've updated it to be `final anusvāra variants (Malayālam, etc.)`, which you can enable for a specific manuscript by adding `<scriptNote xml:id="svarna-nasals"/>` to that file.

I've hesitated normalizing all final nasals because of the potential difference between `kuṣṭān tagaraṃ`  (masculine plural, accusative) and `kuṣṭāṃ tagaraṃ` (feminine singular, accusative). Although in my newer system I've just given up on this and normalize all final nasals... what do you think?

**Comment by wujastyk** [comment]
Thank you, that manuscript-specific tag solves the immediate problem.   I quite like the manuscript-specific (or even scribe specific! ) tagging.  It seems to me to reflect the reality of scribal variation.   And I get the importance of the n-t /  ṃ-t distinction.  I think it's right to make it a MS-specific `<textlang>` switch.

**Comment by wujastyk** [comment]
Thinking more about this, and feeling greedy, can we have both?  The `<scriptNote>` and the ` final anusvāra variants (Malayālam)`?  The latter helps when *all* the MSS behave the same.

**Comment by wujastyk** [comment]
>  `<scriptNote xml:id="svarna-nasals"/>` 

That's `savarna-nasals`?

**Comment by chchch** [comment]
Yes, `savarna-nasals` (oops). I think the current behaviour is as you say — the filter is activated when

1. the checkbox is clicked, and
2. the manuscript has either the `<scriptNote>` tag or the the `@mainlang` set to `sa-Mlym`.
